### ISSUE TITLE
ex vivo DKI reconstruction issues

### ISSUE THREAD

**Issue by andebag** [dipy/dipy#2026]
Hello,

I'm trying to reconstruct some kurtosis maps for ex vivo rodent brains using the dipy dki_mirco processing, but it seems to be zeroing out kurtosis values for much of the brain. Running the standard DTI processing with dipy gives me normal results from other processing techniques. I have tested the example in vivo human brain datasets and got the right results, so I know the code is working correctly. Is there a preprocessing that needs to be done or a variable specified/changed for dki_micro to run properly for ex vivo data, as is the case with NODDI processing?

Thanks!

**Comment by skoudoro** [comment]
Hi @andebag,

Thank you for asking, I just tagged @RafaelNH who develop the DKI module, and he will get back to you as soon as he can.

**Comment by andebag** [comment]
@skoudoro thank you!

I also forgot to mention, the axial kurtosis generated seems to have the most complete reconstruction, but nearly 50% of the brain is still zeroed out. The radial (and mean) kurtosis map is the one nearly all zeroed out.

Ande

**Comment by andebag** [comment]
Hi @RafaelNH 

Just wanted to check and see if you had a chance to check out the issue I had with ex vivo DKI processing? Let me know if I can provide any info you might need. Thanks again!

Ande

**Comment by RafaelNH** [comment]
Hi @andebag! Sorry for the late reply. I have experience in processing ex vivo rodent brain data. That type of problem is typically related to data acquisition than a code issue. 

What MRI system that you used to acquire your data? What are your acquired b-values and gradient directions? How are you acquiring your diffusion-weighted data - with a unique experiment containing all b-values and gradient directions or multiple experiments? Is the receiver gain constant across your experiments?

**Comment by RafaelNH** [comment]
PS: @skoudoro  - why did you unassigned me on this issue?

**Comment by skoudoro** [comment]
I did not, I do not what happens, that's weird... I should check other issue...

**Comment by andebag** [comment]
Hi @RafaelNH ! Not a problem. The data was acquired on a Bruker system at 21 T. My b-values were 0, 1000, 2000, 3000, 4000, 5000 with 45 directions each. For this particular dataset, it was all in one unique dataset. Though, I tried it on a different dataset that had different b-values in separate scans which I combined afterwards, and it yielded the same result. And yes, the RG is constant for all of them. I've attached a google link to a zip file (too big to attach straight to here) that contains my raw data, mask, bvals and bvecs files, and the results of my kurtosis processing is in the DKI folder.

https://drive.google.com/file/d/1AyBURAnNzwtraQXrbwHRcfbs8SIRkTvF/view?usp=sharing

I feel that the data acquisition is fine, considering the standard DTI results were decent. What are your thoughts?
